app/views/images/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :head_script do %>
-<%= javascript_include_tag 'lightbox' %>
+<% javascript_include_tag 'openseadragon' %>
```

Comment:
Could we put this in `:foot_script` instead?  This will keep it consistent with changes in PR#94.